azext_iot/digitaltwins/params.py

```diff
@@ -525,3 +525,56 @@ def load_digitaltwins_arguments(self, _):
             options_list=['--yes', '-y'],
             help='Do not prompt for confirmation when assigning required roles.',
         )
+
+    with self.argument_context("dt job import") as context:
+        context.argument(
+            "job_id",
+            options_list=["--job-id", "-j"],
+            help="ID of bulk import job",
+            arg_group="Bulk Import Job",
+        )
+
+        context.argument(
+            "data_file_name",
+            options_list=["--data-file-name", "--df"],
+            help="Name of data file input to the bulk import job. The file must be in 'ndjson' format.",
+            arg_group="Bulk Import Job",
+        )
+
+        context.argument(
+            "input_blob_container_name",
+            options_list=["--input-blob-container-name", "--ibc"],
```

Comment:
Independent of past precedence, what are your thoughts about dropping the `-name` segment from the newly introduced user facing params/args that have it in the `dt job import` space? The help descriptions mention 'Name of...' across the board, and wondering if the longer `-name` is worth keeping.

Reply:
I agree - do not like the -name ending

Reply:
Makes sense, will remove the **-name** segment